test/shard.js

```diff
@@ -20,8 +20,8 @@ client.on('message', msg => {
 process.send(123);
 
 client.on('ready', () => {
-  console.log('Ready', client.options.shardId);
-  if (client.options.shardId === 0)
+  console.log('Ready', client.options.shardID);
+  if (client.options.shardID === 0)
     setTimeout(() => {
       console.log('kek dying');
```

Comment:
thonk

Reply:
i guess for testing it doesn't matter.  😆

Reply:
The option was renamed to shardID so the test needs to be updated aswell